source/frontend/src/features/mapSideBar/acquisition/models/PayeeOptionModel.ts

```diff
@@ -223,6 +251,42 @@ export class PayeeOption {
     );
   }
 
+  public static createLeaseStakeholder(model: ApiGen_Concepts_LeaseStakeholder): PayeeOption {
+    let payeeName: string;
+    let payeeDescription: string;
+
+    switch (model.lessorType.id) {
+      case ApiGen_CodeTypes_LessorTypes.ORG:
+        payeeName = `${model.organization.name}, Inc. No. ${model.organization.incorporationNumber}`;
```

Comment:
I don't think incorporation number is required, this logic may need to handle not displaying Inc. No. in that case.

Reply:
I am little hesitant to modify this since I am using previously developed and tested logic.